easybuild/easyblocks/c/cp2k.py

```diff
@@ -103,14 +105,16 @@ def extra_options():
             'plumed': [None, "Enable PLUMED support", CUSTOM],
             'type': ['popt', "Type of build ('popt' or 'psmp')", CUSTOM],
             'typeopt': [True, "Enable optimization", CUSTOM],
+            'v2023': [False, "2023 version of CP2K", CUSTOM],
```

Comment:
This should not be a config option, use LooseVersion(get_software_version('CP2K')) to figure out if it is 2023 or newer instead.